lib/source_wms.c

```diff
@@ -92,7 +93,7 @@ void _mapcache_source_wms_render_map(mapcache_context *ctx, mapcache_map *map)
   mapcache_http_do_request(ctx,http,map->encoded_data,NULL,NULL);
   GC_CHECK_ERROR(ctx);
 
-  if(!mapcache_imageio_is_valid_format(ctx,map->encoded_data)) {
+  if(map->tileset->format->type != GC_RAW && !mapcache_imageio_is_valid_format(ctx,map->encoded_data)) {
```

Comment:
not checking tileset->format!=NULL might be the cause of the crash in https://github.com/mapserver/mapserver/issues/5596#issuecomment-454412529_